nnpdf_data/nnpdf_data/commondata/NNPDF_POS_3P16GEV/metadata.yaml

```diff
@@ -0,0 +1,245 @@
+setname: NNPDF_POS_3P16GEV
+version: 1
+version_comment: POS at Q2 = 10 GeV^2
+nnpdf_metadata:
+  nnpdf31_process: POSITIVITY
+  experiment: NNPDF
+arXiv:
+  url: https://arxiv.org/abs/
+iNSPIRE:
+  url: ''
+hepdata:
+  url: ''
+  version: -1
+implemented_observables:
+- observable_name: XUQ
+  observable:
+    description: Deep Inelastic Scattering
+    label: 'positivity dataset: $xu$ PDF $Q^2=10$ GeV$^2$'
+    units: ''
+  process_type: POS_XPDF
+  tables: []
+  npoints: []
+  ndata: 20
+  plotting:
+    theory_reference: Bertone:2013vaa
```

Comment:
actually why do we have such a key? and can we please change it to
```bibtex
@article{Candido:2024rkr,
    author = "Candido, Alessandro and Hekhorn, Felix and Magni, Giacomo and Rabemananjara, Tanjona R. and Stegeman, Roy",
    title = "{Yadism: yet another deep-inelastic scattering module}",
    eprint = "2401.15187",
    archivePrefix = "arXiv",
    primaryClass = "hep-ph",
    reportNumber = "CERN-TH-2024-015, Edinburgh 2024/4, Nikhef-2024-002",
    doi = "10.1140/epjc/s10052-024-12972-7",
    journal = "Eur. Phys. J. C",
    volume = "84",
    number = "7",
    pages = "698",
    year = "2024"
}
```

Reply:
> they can be removed

I think that would be better - because data can not know which "theory" was used (nor which generator nor which settings etc.)

Reply:
So remove`theory_reference` key within plotting?

Reply:
Yes. If it is fine I'll add a commit removing it from everywhere. It's only left around because of the ported data.